common/libradoc/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
```

Comment:
nit: since this is not really a prod dependency, would it make sense to move this to a different folder? (e.g. tooling/, specs/, testsuite/)

Reply:
I don't mind as long as such directory exists already. Note that I see some proptest helpers and workspace hack in `common/`.

Reply:
I wish the distinction between prod stuff and dev stuff was clearer FWIW